react-debugging.html

```diff
@@ -8,84 +8,89 @@
 
```

Comment:
Looks like some things got changed...but they look the same. I do see the changes that were needed to get the cards to appear. React fragments, and properly calling the functions in React. Good work!